app/views/projects/dataset/edit.html.erb

```diff
@@ -0,0 +1,46 @@
+<% content_for :javascript_head do %>
+  <script type="text/javascript" src="https://unpkg.com/trix@2.0.8/dist/trix.umd.min.js"></script>
+<% end %>
+
+<div id="project-form" class="inner-canvas">
+  <div class="inner-canvas-header">
+    <div class="row">
+      <% @title = _("Edit your dataset") %>
+      <h1><%= @title %></h1>
+    </div>
+  </div>
+
+  <div class="inner-canvas-body">
+    <div class="row">
+      <%= foi_error_messages_for :project, full_message: true %>
+    </div>
+
+    <div class="row">
+      <%= form_for @project, url: project_dataset_path(@project) do |f| %>
+        <div class="form_input">
+          <%= f.label :dataset_description, _('Description'), class: 'form_label' %>
+          <%= f.rich_text_area :dataset_description %>
+        </div>
+
+        <div class="form_input">
+          <%= f.label :dataset_public, _('Visibility'), class: 'form_label' %>
+
+          <div class="radio_buttons">
```

Comment:
Since the default is "Private", think it would be best to swap the order here so that the Private radio button appears first (and is selected by default), as (at least for most alavetelis) the ordering of left to right is more natural.

At present, the default state feels a bit weird. It feels like the right-most radio button is the "option" rather than the "default".

![image](https://github.com/user-attachments/assets/eae7dcb4-4eb1-426d-bec4-da2a33b0afe0)

I'd expect this, where the selected default appears first, with the option value second and unselected by default.

![image](https://github.com/user-attachments/assets/e6821a88-fe17-462f-a4bf-5c9486dd56d5)